openshift_cli_installer/libs/clusters/ocp_cluster.py

```diff
@@ -48,8 +48,12 @@ def __init__(self, ocp_cluster, user_input):
         self.logger = get_logger(f"{self.__class__.__module__}-{self.__class__.__name__}")
         self.cluster = ocp_cluster
 
-        self.s3_bucket_name = self.user_input.s3_bucket_name or self.cluster["cluster_info"].get("s3_bucket_name")
-        self.s3_bucket_path = self.user_input.s3_bucket_path or self.cluster["cluster_info"].get("s3_bucket_path")
+        self.s3_bucket_name = self.user_input.s3_bucket_name or (
+            self.cluster["cluster_info"].get("s3_bucket_name") if self.cluster.get("cluster_info") else None
```

Comment:
why is this change needed?

Reply:
`self.cluster["cluster_info"]` does not exist on cluster create.
It gets written to cluster data yaml file on init (`cluster_info` not in [keys_to_pop](https://github.com/RedHatQE/openshift-cli-installer/blob/dfabe4706b7cbd891d3cc73bd8b52943e772f66f/openshift_cli_installer/libs/clusters/ocp_cluster.py#L189))